Sources/App/entrypoint.swift

```diff
@@ -1,21 +1,25 @@
 import Vapor
 import Logging
+import NIOCore
+import NIOPosix
 
 @main
 enum Entrypoint {
     static func main() async throws {
+        _ = NIOSingletons.unsafeTryInstallSingletonPosixEventLoopGroupAsConcurrencyGlobalExecutor()
```

Comment:
```suggestion
        let success = NIOSingletons.unsafeTryInstallSingletonPosixEventLoopGroupAsConcurrencyGlobalExecutor()
```

I'd probably log at debug message or so whether installing NIO as the global concurrency executor worked or not.